hack/stabilization-changes.py

```diff
@@ -128,16 +138,21 @@ def stabilize_release(version, channel_name, channel_path, delay, errata, feeder
         else:
             yield '{}. {} {}'.format(subject, body, pull.html_url)
     else:
-        _LOGGER.info('  waiting: {} ({}){}'.format(version, version_delay, public_errata_message))
+        _LOGGER.info('  waiting: {} ({}){}'.format(version, merged_ago, public_errata_message))
         if waiting_notifications:
-            yield 'Recommend waiting to promote {} to {}; it was promoted the feeder {} by {} ({}, {}, {}){}'.format(
+            # Show either expected promotion time or when
+            time_message = None
+            if delay:
+                # Show expected date to promote
+                time_message = "Expected to be promoted on {}".format(feeder_promotion['committer-time'] + delay)
```

Comment:
I'm fine with this, but "expected" might be too strong.  Perhaps, `Current promotion delay will be satisfied on {}` or something that is more clearly "I'm just talking about my current config" and has less likelihood of being treated as a prediction of future activity (because the config could change, or we could [promote by hand without waiting out the delay][1], etc.).

[1]: https://github.com/openshift/cincinnati-graph-data/pull/1134/files#diff-4fda33ea6ee80e385cd006122ed0861e642f546da009afc6a3c8d38ef226d233R7

Reply:
Good idea, reworked in 063f073faa0bacb073757f6d35e135bc40b19cfd